docs/topics/auth/passwords.txt

```diff
@@ -236,3 +236,173 @@ from the ``User`` model.
+By default, validators are used in the forms used to reset or change
+passwords. The default for setting is an empty list, which means no validators
+are applied. In new projects created with the default :djadmin:`startproject`,
+a simple set of validators is enabled.
+
+
+Enabling password validation
+----------------------------
+Password validation is configured in the
+:setting:`AUTH_PASSWORD_VALIDATORS` setting::
+
+    AUTH_PASSWORD_VALIDATORS = [
+        {
+            'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
+        },
+        {
+            'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
+            'OPTIONS': {
+                'min_length': 9,
+            }
+        },
+        {
+            'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
+        },
+    ]
+
+This example enables three validators:
+
+* ``UserAttributeSimilarityValidator``, which checks the similarity between
+  the password and a set of attributes of the user.
+* ``MinimumLengthValidator``, which simply checks whether the password meets a
+  minimum length. This validator is configured with a custom option: it now
+  requires the minimum length to be nine characters, instead of the default
+  eight.
+* ``CommonPasswordValidator``, which compares the password to a list of 1000
+  most common passwords.
+
+For ``UserAttributeSimilarityValidator`` and ``CommonPasswordValidator``,
+we're simply using the default settings.
+
+The help texts and any errors from password validators are always returned in
+the order they are listed in :setting:`AUTH_PASSWORD_VALIDATORS`.
+
+
+Included validators
+-------------------
+Django includes three validators:
+
+.. class:: MinimumLengthValidator(min_length=8)
+
+    Validates whether the password meets a minimum length.
+    The minimum length can be customized with the ``min_length`` parameter.
+
+.. class:: UserAttributeSimilarityValidator(user_attributes=None, max_similarity=0.7)
+
+    Validates whether the password is sufficiently different from certain
+    attributes of the user.
+
+    The ``user_attributes`` parameter should be an iterable of names of user
+    attributes to compare to. If this is ``None``, the default is used:
+    ``'username', 'first_name', 'last_name', 'email'``.
+    Attributes that don't exist are ignored.
+
+    The maximum similarity the password can have, before it is rejected, can
+    be set with the ``max_similarity`` parameter, on a scale of 0 to 1.
+    A setting of 0 will cause all passwords to be rejected, whereas a setting
+    of 1 will cause it to only rejects passwords that are identical to an
+    attribute's value.
+
+
+.. class:: CommonPasswordValidator()
+
+    Validates whether the password is not a common password, by checking
+    against a list of 1000 most common password created by
+    `Mark Burnett <https://xato.net/passwords/more-top-worst-passwords/>`_.
```

Comment:
I've seen somewhat recent references to it, but it still fails for me. Should we link to the [archived version](https://web.archive.org/web/20150315154609/https://xato.net/passwords/more-top-worst-passwords/)? Attribution does appear to be required.